docs/developer/base/logs.md

```diff
@@ -0,0 +1,18 @@
+# Logs
+
+::: datadog_checks.base.checks.logs.crawler.stream.LogRecord
+    options:
+      heading_level: 2
+      members:
+        - cursor
+        - data
+
+::: datadog_checks.base.checks.logs.crawler.stream.LogStream
+    options:
+      heading_level: 2
+      filters: ["!^_[^_]?", "^__.+__$"]
```

Comment:
Please only expose individual members, and below

Reply:
👍